paddle/fluid/framework/ir/mkldnn/cpu_bfloat16_pass.cc

```diff
@@ -169,60 +169,134 @@ void AddReoderBeforeSingleInputs(ir::Graph* graph, int* quantize_counter) {
 
 void CPUBFloat16Pass::SetInputDataType(ir::Graph* graph) const {
   int quantize_counter = 0;
-  AddReoderBeforeDuplicatedInputs(graph, &quantize_counter);
-  AddReoderBeforeSingleInputs(graph, &quantize_counter);
+  AddReoderBeforeDuplicatedInputs(graph, quantize_counter);
+  AddReoderBeforeSingleInputs(graph, quantize_counter);
   PrettyLogDetail("---    added %d quantize ops before bfloat16 op",
                   quantize_counter);
 }
 
-void CPUBFloat16Pass::SetOutputDataType(ir::Graph* graph) const {
+void AddDequantize(Graph* g, ir::Node* op, ir::Node* op_out,
+                   int& dequantize_counter) {
+  if (op->Op()->Type() == "prior_box") return;
```

Comment:
Why does prior_box return directly? Could you add descriptions or can we use sets to maintain operators that require special handling?

Reply:
Prior_box output always produces floating-point results because these are prior boxes generated. Therefore, we do not need dequantization.

Reply:
Jakub may add comments in next PR, but this PR has passed all CIs and hopefully it could be merged